src/common/util.test.ts

```diff
@@ -120,3 +121,14 @@ it('optionHasValue works correctly', () => {
   assert.equal(optionHasValue(some({foo: {bar: 'wrong'}}), {foo: {bar: 'baz'}}), false)
   assert.equal(optionHasValue(none, 1), false)
 })
+
+it('pads short private key to the correct length', () => {
```

Comment:
Another important test wrt padding - derived address/public key is the same with padded and unpadded version.

Reply:
My findings:
- `web3.eth.accounts.privateKeyToAccount` doesn't accept the non-padded PK.
- The method used in `walletState/addAccount` (`web3.eth.accounts.encrypt`) doesn't complain, but it doesn't produce the same address for padded vs non-padded PKs.

Surprisingly, using `encrypt` with non-padded and send tx with the padded PK results in correct calls, the transactions go through. 

Padding the PK at the time of restoring the wallet results in a different account. When sending tx, the used padded private key is the same for the two different accounts (different addresses), but sending works for both! I'm not sure how it's possible.